content/press.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: "Jenkins Press Information"
+---
+
+
+If you're a journalist, blogger or anybody else who might consider themselves
+"press" this page should help you refer to, explain or otherwise cite the
+Jenkins project correctly. If this page does not sufficiently answer a question
+you may have, or you're seeking comment, please reach out to one of our <<Press Contacts>> below.
+
+
+== About Jenkins
+
+
+.Elevator Pitch
+""
+Jenkins is an open source automation server which enables developers around the
+world reliably build, test and deploy their software
+""
+
+.Blurb
+""
+Jenkins, originally founded in 2006 as "Hudson", is one of the leading
+automation servers available. Using an extensible, plugin-based,
+architecture developers have created hundreds of plugins to adapt Jenkins to a
+multitude of build, test, and deployment automation workloads. In 2015, Jenkins
+surpassed 100,000 known Jenkins installations making it the most
+widely deployed automation server.
+""
+
+
+=== Governance
+
+The
+link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document[Governance
+Document] which outlines the project's philosophy, structure and guidelines is
+all publicly available. This also includes information on the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Board[Governance
+Board], which is the group with acting executive authority. While board members
+are available for comment, we encourage you to reach out to our <<Press Contacts>>
+as an initial point of contact for press inquiries.
+
+
+The Jenkins project is part of link:http://spi-inc.org/[Software in the Public
+Interest], a 501 (c) (3) non-profit organization, which holds the Jenkins
+trademark and acts as a legal umbrella organization.
+
+=== Branding
+
+We ask that the following branding guidelines be used when referring to or
+writing about the Jenkins project:
+
+*Correct:*
+
+* the Jenkins project
+* Jenkins
+
+*Incorrect:*
+
+* Jenkins CI
+* Jenkins server
+* Mr. Jenkins
```

Comment:
Branding section! :+1: 

When the new website lands, we'll need to have some sort of crazy contest to get people to find all the instances of "Jenkins CI" we have that need to be changed :)   Starting with the Twitter bio!